.appveyor.yml

```diff
@@ -134,10 +134,13 @@ matrix:
       CMAKE_PREFIX_PATH: "C:\\mingw-w64\\x86_64-7.2.0-posix-seh-rt_v5-rev1"
 
 before_build:
-  - set Path=C:\\MinGW\\bin;%Path%
+  # See https://help.appveyor.com/discussions/problems/3193-cmake-building-for-mingw-issue-with-git-shexe
+  - set PATH=%PATH:C:\Program Files (x86)\Git\bin;=%
```

Comment:
Looks like this is more elegant solution as my original which renamed `sh.exe`.

Similar problem I had with building C module for Perl as on AppVeyor are 3 different perl versions and 2 different MinGW versions which Perl can detect. And I just renamed binaries...:
https://github.com/gooddata/DBD-MariaDB/commit/97d28bb3cf05c952c5784ed9ed1a33b06df262d6#diff-180360612c6b8c4ed830919bbb4dd459R36